src/backend/main/py.main/caliopen_main/user/store/user.py

```diff
@@ -74,11 +74,12 @@ class RemoteIdentity(BaseModel):
     """User remote identities model."""
 
     user_id = columns.UUID(primary_key=True)
-    identity_id = columns.Text(primary_key=True)
+    identifier = columns.Text(primary_key=True)
+    display_name = columns.Text()
     type = columns.Text()
     status = columns.Text()
-    credentials = columns.List(columns.Text)
     last_check = columns.DateTime()
+    infos = columns.Map(columns.Text, columns.Text)
```

Comment:
username/password is not an universal scheme (could be a client_secret and client_token on an oauth authentication system), so I prefer to be agnostic about informations for collecting a remote identity. informations stored are dependent of remote identity type (server, port, username, password for an imap).

About storing this credentials encrypted, I agree completely. But we must define how to do (which encryption/decryption algorithm, how to decrypt, etc). For development purposes this is not needed now and we can enhance later with encryption when we have define how to proceed